src/main/java/no/fdk/service/AssessmentService.java

```diff
@@ -110,7 +110,10 @@
         ExampleMatcher matcher = ExampleMatcher.matching().withIgnoreNullValues();
         Example<Assessment> example = Example.of(assessment, matcher);
 
-        Flux<Assessment> assessments = assessmentRepository.findAll(example);
+        Flux<Assessment> assessments = assessmentRepository
```

Comment:
Testet litt i staging nå
Maksimal verdi på `totalElements` er nå lik `size`, så det ser ut som om alle har 10 eller mindre datasett

Reply:
Bra spotted! Pushet fiksen nu

Reply:
Nå viser den totalt antall fra alle publishers, dvs alltid 1537

Reply:
Sjekk igjen om par min, da skal begge issues være løst

Reply:
Nå sier den 19, ikke 20, for Liland, men det er vel noe annet